test_base_exception/tests/test_base_exception.py

```diff
@@ -17,29 +14,10 @@
 class TestBaseException(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
-        super(TestBaseException, cls).setUpClass()
-        cls.loader = FakeModelLoader(cls.env, cls.__module__)
-        cls.loader.backup_registry()
-
-        # The fake class is imported here !! After the backup_registry
-        from .purchase_test import LineTest, PurchaseTest
-
-        cls.loader.update_registry((LineTest, PurchaseTest))
+        super().setUpClass()
 
         cls.base_exception = cls.env["base.exception"]
         cls.exception_rule = cls.env["exception.rule"]
-        if "test_purchase_ids" not in cls.exception_rule._fields:
-            field = fields.Many2many("base.exception.test.purchase")
-            cls.exception_rule._add_field("test_purchase_ids", field)
-            cls.exception_rule._fields["test_purchase_ids"].depends_context = None
-        cls.exception_confirm = cls.env["exception.rule.confirm"]
-        cls.exception_rule._fields["model"].selection.append(
-            ("base.exception.test.purchase", "Purchase Order")
-        )
-
-        cls.exception_rule._fields["model"].selection.append(
-            ("base.exception.test.purchase.line", "Purchase Order Line")
```

Comment:
do not remove tests

Reply:
I did a new PR https://github.com/OCA/server-tools/pull/2366 that solves this problem